openstack_controller/tests/test_simple_api.py

```diff
@@ -664,6 +671,53 @@ def test__get_paginated_list():
         result = api._get_paginated_list("url", "obj", {})
         assert log.debug.call_count == 1
         assert result == [{"id": 0}, {"id": 1}, {"id": 2}, {"id": 3}, {"id": 4}]
+        log.reset_mock()
+
+    with mock.patch(
+        "datadog_checks.openstack_controller.api.SimpleApi._make_request",
+        side_effect=[
+            # Fourth call: 1 AuthenticationNeeded exception -> no retries
+            AuthenticationNeeded,
+            # Fifth call: any other exception -> no retries
+            Exception,
+        ]
+    ):
+        with pytest.raises(AuthenticationNeeded):
+            api._get_paginated_list("url", "obj", {})
+        with pytest.raises(Exception):
+            api._get_paginated_list("url", "obj", {})
+
+
+def test__make_request_failure():
```

Comment:
double `_`

Reply:
on purpose: the name of the method is `_make_request`